packages/blockchain-tools/src/chains/BlockchainManager.ts

```diff
@@ -84,8 +87,10 @@ const BlockchainManager = {
     server: new Web3Server("xdc", getNetwork("xdc")),
   },
   xrpl: {
+    //client: new CrossmarkWallet("xrpl", getNetwork("xrpl")),
+    //client: new GemWallet("xrpl", getNetwork("xrpl")),
     client: new XummClient("xrpl", getNetwork("xrpl")),
-    server: new XrplServer("xrpl", getNetwork("xrpl")),
+    server: new XrplServer("xrpl", getNetwork("xrpl"), 77777777), // NFT TAG SHOULD BE MOVED TO MINT_NFT METHOD
```

Comment:
Good idea. Any reason not to?

Reply:
Not urgent, but keep in mind for future refactoring